core/modules/tests/SConscript

```diff
@@ -2,4 +2,40 @@
 Import('env')
 Import('standardModule')
 
-standardModule(env, unit_tests="")
+import os.path
+
+# Harvest special binary products - files starting with the package's name
+# followed by underscore:
+#
+#   tests_<something>.cc
+#
+# Then make a Scons target for each of those to be returned by the calling
+# SConscript. Also add the source file into a list of sources to be excluded
+# from the shared library product.
+
+env.Append(LIBPATH=['$build_dir'] + env['LIBPATH'])
```

Comment:
That changes global environment which you don't want to do.

Reply:
Should I then pass this as a parameter to the Program target when doing?
 env.Program(...LIBPATH=['$build_dir'] + env['LIBPATH']...)

Reply:
Okay, I've deleted a line of code which modified LIBPATH in the global environment and passed it to each env.Program() invocation as a parameter. And this works.